commentary/scope-sets.md

```diff
@@ -0,0 +1,116 @@
+A "scope" intuitively means a region of code where some variables are visible. In a language without hygienic macros, scopes are neatly nested within each other, and when multiple bindings bind the same name, the innermost binding wins:
+```
+-- example 1
```

Comment:
```suggestion
-- example 1: This example demonstrates scope sets in a language that does not have hygenic macros
```

Reply:
```suggestion
-- example 1: This example demonstrates scopes in a language that does not have macros
```
your version made it sound like the language has unhygienic macros, but really, this is just an example which does not use macros